### DISCUSSION TITLE
Issues in Android studio 4.2 beta 6

### DISCUSSION THREAD

**Discussion by maanuanubhav999** [google/android-fhir#382] [General]
I was trying to set up beta version for this project but struck with some errors.  [link to stackoverflow](https://stackoverflow.com/questions/66746453/android-studio-4-2-beta-6-gradle-issue)

Anyone else facing similar issues ? 
Possible solutions.

**Comment by s-ayush2903** [comment]
(An excerpt from the SO question mentioned above)
_Re-download dependencies and sync project (requires network)
The state of a Gradle build process (daemon) may be corrupt. Stopping all Gradle daemons may solve this problem._

This hints me that you have offline sync enabled in your android studio. You should change it to online sync so that it can download the required deps and libs

**Reply by s-ayush2903** [reply]
Refer the following answer to toggle that setting https://stackoverflow.com/a/59194206 . If this doesn't work then it'll be quite interesting to figure it out. :wink:

**Reply by maanuanubhav999** [reply]
hello, thanks for the help.
I tried above mentioned but issue still persists.

**Reply by maanuanubhav999** [reply]
Trying again by restoring all settings etc. will let you know.

**Comment by Ana2k** [comment]
> I was trying to set up beta version for this project but struck with some errors. [link to stackoverflow](https://stackoverflow.com/questions/66746453/android-studio-4-2-beta-6-gradle-issue)
> 
> Anyone else facing similar issues ?
> Possible solutions.

Hi I hope I am not replying to late?
The android version conflicts was creating a lot of error for me too. 
You can try changing the deps.gradle file temporarily to a previous stable version and working on it for a while. Feel free to reach out if you need further help :)

**Reply by maanuanubhav999** [reply]
hello, thanks for replying. Currently, the issue has been resolved through some trial and error but will surely keep this in mind.
